corehq/apps/reports/models.py

```diff
@@ -939,17 +939,6 @@ def uses_cases(self):
         return False
 
 
-class FormDeidExportSchema(FormExportSchema):
```

Comment:
@dannyroberts can you confirm there's nothing magic that needs this and would somehow use it in a way that doesn't pass the grep test? guessing not, just being paranoid since it's exports...